onnx/defs/nn/defs.cc

```diff
@@ -101,20 +101,12 @@ void convPoolShapeInference(
     if ((nullptr != auto_pad_attr) && (auto_pad_attr->s() != "VALID")) {
```

Comment:
it seems the check for ```auto_pad_attr->s() != "NOTSET"``` was removed in the previous PR. One edge case slipped through that when auto_pad=NOTSET, and pads are not provided. This doesn't affect the outcome though, as pads values will remain zero for NOTSET.

Reply:
I think I did make this comment in the original PR - https://github.com/onnx/onnx/pull/1988